examples/streaming_multitask/streaming_multitask.py

```diff
@@ -16,14 +16,14 @@ class User(BaseModel):
     age: int
 
 
-def stream_extract(input: str, cls) -> Iterable[User]:
-    MultiUser = instructor.MultiTask(cls)
-    completion = client.chat.completions.create(
+Users = Iterable[User]
+
+def stream_extract(input: str) -> Iterable[User]:
```

Comment:
```suggestion
def stream_extract(input: str) -> Users
```

this would work too right? or even `response_model=Iterable[User]`

love it

Reply:
yup that works